File: TunesClient/TunesClient/Tunes.cs
using System;
using System.Runtime.Serialization;

namespace TunesClient
{
    [DataContract]
    class Tunes
    {
        [DataMember(Name = "id")]
        public int Id { get; set; }
        [DataMember(Name = "irishChart")]
        public int IrishChart { get; set; }
        [DataMember(Name = "title")]
        public String Title { get; set; }
        [DataMember(Name = "artist")]
        public String Artist { get; set; }
        [DataMember(Name = "album")]
        public String Album { get; set; }
        [DataMember(Name = "genre")]
        public String Genre { get; set; }
        [DataMember(Name = "albumCoverLink")]
        public String AlbumCoverLink { get; set; }
        [DataMember(Name = "price")]
        public double Price { get; set; }
        [DataMember(Name = "buyLink")]
        public String BuyLink { get; set; }
        [DataMember(Name = "release")]
        public DateTime Realsed { get; set; }
        [DataMember(Name = "duration")]
        public String Duration { get; set; }

        public override string ToString()
        {
            return IrishChart + "\t" + Title + "\t" + Artist + "\t" + Album + "\t" + Genre;
        }
    }

    [DataContract]
    class SuggestedTunes
    {
        [DataMember(Name = "id")]
        public int Id { get; set; }
        [DataMember(Name = "count")]
        public int Count { get; set; }
        [DataMember(Name = "title")]
        public String Title { get; set; }
        [DataMember(Name = "artist")]
        public String Artist { get; set; }
        [DataMember(Name = "genre")]
        public String Genre { get; set; }

        public override string ToString()
        {
            return Count + "\t" + Title + "\t" + Artist + "\t" + Genre;
        }
    }
}

File: trial/app/src/main/java/com/example/trial/Tunes.java
package com.example.trial;

import com.google.gson.annotations.SerializedName;

public class Tunes {
    @SerializedName("id")
    public int ID;
    @SerializedName("irishChart")
    public int IrishChart;
    @SerializedName("title")
    public String Title;
    @SerializedName("artist")
    public String Artist;
    @SerializedName("album")
    public String Album;
    @SerializedName("genre")
    public String Genre;
    @SerializedName("albumCoverLink")
    public String AlbumCoverLink;
    @SerializedName("price")
    public double Price;
    @SerializedName("buyLink")
    public String BuyLink;
    @SerializedName("realsed")
    public String Realsed;
    @SerializedName("duration")
    public String Duration;

    public Tunes() {

    }


   public Tunes(String artist, String title, String album, String url) {
        this.Artist = artist;
        this.Title = title;
        this.Album = album;
        this.AlbumCoverLink = url;
    }


    public int getID() {
        return ID;
    }

    public String getArtist() {
        return Artist;
    }

    public String getTitle() {
        return Title;
    }

    public String getAlbumCoverLink() {
        return AlbumCoverLink;
    }

    public String getAlbum() {
        return Album;
    }

    public String getGenre() {
        return Genre;
    }


    public String getDuration() {
        return Duration;
    }
    public String getBuyLink() {
        return BuyLink;
    }

    public String toString() {
        return Title;
    }

}

class BuyAlbums extends Tunes
{
    public BuyAlbums(String artist, String album, String albumCoverLink, String buy) {
        this.Artist = artist;
        this.Album = album;
        this.AlbumCoverLink = albumCoverLink;
        this.BuyLink = buy;
    }

}




/*
class Stats extends Tunes {
    public String toString() {
        return getDuration();
    }
}

class GenreInfo extends Tunes {
    public String toString() {
        return  "\n" + getArtist() + "\t" + getTitle() + "\t" + getGenre() ;
    }
} */


